includes/wikia/helpers/TemplatePageHelper.php

```diff
@@ -125,6 +125,30 @@ public function getTemplateParams() {
 			}
 		}
 
+		$infoboxes = PortableInfoboxDataService::newFromTitle( $this->getTitle() )->getData();
+		if ( !empty( $infoboxes ) ) {
+			foreach ( $infoboxes as $infobox ) {
+				if ( isset( $infobox['sourcelabels'] ) ) {
+					// TODO remove after XW-2415 is merged
+					foreach ( $infobox['sourcelabels'] as $sourceName => $sourceLabel ) {
+						if ( !isset( $params[$sourceName] ) ) {
+							$params[$sourceName] = 1;
+						}
+					}
+				} elseif ( isset( $infobox['metadata'] ) ) {
+					// TODO run only this one after XW-2415 is merged
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)